runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -573,24 +573,20 @@ Table<K, V, M> getTable(String namespace, String tableName) {
     }
 
     /**
-     * Close a table that is already opened.
+     * Allow underlying objects of this table to be garbage collected
+     * while still retaining the metadta of the table.
      *
      * @param namespace Namespace of the table.
      * @param tableName Name of the table.
      * @throws NoSuchElementException - if the table does not exist.
      */
-    public void closeTable(String namespace, String tableName) {
+    public void freeTableData(String namespace, String tableName) {
         String fullyQualifiedTableName = getFullyQualifiedTableName(namespace, tableName);
         Table<Message, Message, Message> table = tableMap.get(fullyQualifiedTableName);
         if (table == null) {
             throw new NoSuchElementException("closeTable: Did not find any table "+ fullyQualifiedTableName);
```

Comment:
nit -> closeTable -> freeTableData in exception message

Reply:
done, thanks for catching this.